docs/litecoin.md

```diff
@@ -1,43 +1,99 @@
 # ckb-auth litecoin interoperability
 
 Ckb-auth library is able to verify a lot of blockchain signatures including the litecoin.
-We can use `litecoin-cli` (or other compatible wallet) to generate valid transactions and validate them on-chain with ckb-auth.
+We can use `litecoin-cli` (or other compatible wallet) to generate valid signatures and validate them on-chain with ckb-auth.
 
-We need to sign the message with your litecoin private key. The message to be signed is the ASCII hex representation of
-the transaction hash. This signing process is itself straightforward. But we may to do some serialization/deserialziation
-works for the message, private key and signature.
+A simple way to use litecoin signature algorithm to lock ckb cells
+is to sign the transaction hash (or maybe `sighash_all`, i.e. hashing all fields 
+including transaction hash and other witnesses in this input group)
+with `litecoin-cli`, and then leverage ckb-auth to check the validity of this signature.
+
+# Signing a transaction with litecoin-cli
+
+## Downloading the litecoin binaries
+The commands below download the official `litecoin` binaries (e.g. `litecoind` for litecoin daemon,
+`litecoin-cli` for litecoin command line client) into the directory `/usr/local`.
 
-## Signing the transaction
-In the following section, we assume the private key to signing message is the environment variable `$KEY`
-whose corresponding litecoin address is `$ADDRESS`, and the message to sign is the transaction hash `$MESSAGE`.
-An example of those data is 
 ```
-export KEY="cQoJiU5ECnVpRqfV5dWKDE2sLQq6516Tja1Hb1GABUV24n7WkqV4" ADDRESS="mhknqLHQGWDXuLsPdzab8nA4jD3fMdVYS2" MESSAGE="29553f9e37fa16e45f1d3e616ac5366f6afd9936477f2d6fc870f49bdf540157"
+tarball=litecoin.tar.gz
+wget -O "$tarball" https://download.litecoin.org/litecoin-0.21.2.2/linux/litecoin-0.21.2.2-x86_64-linux-gnu.tar.gz
+tar xvzf "$tarball"
+sudo cp -r litecoin-*/* /usr/local
 ```
 
-### Get the transaction hash
-You need to construct a skeleton transaction and obtain the transaction hash. See [Tutorial | Lumos](https://lumos-website.vercel.app/#5-create-a-transfer-transaction) for a example to do such thing with Lumos.
+## Starting the litecoin daemon
+Litecoin daemon is required in order to for litecoin-cli to work. For example,
+we can run `litecoind -testnet` to start a litecoin testnet node.
```

Comment:
```suggestion
To work with Litecoin in a test environment, you need to start a Litecoin testnet node. You can do this using the `litecoind` command with the `-testnet` option. Here's how:

```bash
litecoind -testnet -daemon
```
```

Reply:
Consider starting a litecoin node in the background